DQMOffline/JetMET/python/jetMETAnalyzer_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-from DQMOffline.JetMET.jptDQMConfig_cff import *      # parameters for jpt analyzer
+#from DQMOffline.JetMET.jptDQMConfig_cff import *      # parameters for jpt analyzer
```

Comment:
Add a comment saying that this line is commented since the config file does not exist